WalletWasabi.Gui/Program.cs

```diff
@@ -73,7 +81,6 @@ private static async void AppMainAsync(string[] args)
 			MainWindowViewModel.Instance = new MainWindowViewModel();
 			StatusBar = new StatusBarViewModel();
```

Comment:
I think you must remove the statusbar stuff too, as you did the same with it as to the lockscreen.

Reply:
I left that because there is some cleanup code for when the application closes. I'll look into it.

Reply:
Fixed